postgres/supabase/supabase.md

```diff
@@ -0,0 +1,20 @@
+### Follow these steps to get started with Supabase as a Data Accelerator.
```

Comment:
Wouldn't the scenario be to use Supabase as a Data Connector not Data Accelerator, correct?

Reply:
This guide as written uses it as an accelerator engine

Reply:
I don't think Data Connector provides as much "wow" factor since the data isn't updated in real time in as exciting of a way. Though we could add another quickstart that follows similar steps for a Data Connector and then instructs the user to make manual inserts into their supabase table and then observe that they are pulled down to their local runtime?